playbooks/platform-uninstall.yml

```diff
@@ -0,0 +1,123 @@
+---
+#### This playbook will uninstall ACS
+
+ - hosts: database
+   tasks:
+     - name: Stopping the database
+       ansible.builtin.service:
+         name: postgresql{% if ansible_os_family != 'Debian' %}{{ dependencies_version.postgres_major_version }}{% endif %}
+         state: stopped
+         enabled: no
```

Comment:
unnecessary as we remove the package later on and the service is deployed with the package